noisyatom/aboutus/views.py

```diff
@@ -0,0 +1,33 @@
+from django.shortcuts import render
+from .models import About, Team
+
+# Create your views here.
+from django.http import HttpResponse
+
+
+# def index(request):
+#    return HttpResponse("Hello, world. You're at the products index.")
+
+def aboutus(request):
+    template_name = 'aboutus.html'
+    aboutus = About.objects.all()
+    team = Team.objects.all()
+
+    for item in aboutus:
+        print(item)
+        print(item.main_title)
+    context = {
+        'main_title': item.main_title,
+        'description1': item.description1,
+        'description2': item.description2,
+        'description3': item.description3,
+        'team': team,
+    }
+    return render(request, template_name, context)
+
+
```

Comment:
Why does the 'team' function render a template called 'aboutus.html'.
This is really confusing, as I would expect the function **aboutus(request)** to render a template called aboutus.html.

Can you maybe call the template something else? Like teams.html ? :-)

Reply:
I see your point. However there are two functions rendered first the 'aboutus' function and then the 'team' function. Would you prefer to split the aboutus.html into two parts?